source/documentation/deploying_services/postgres.md

```diff
@@ -4,49 +4,62 @@ GOV.UK PaaS enables you to create a PostgreSQL database service (powered by Amaz
 
+There is a free plan available with limited storage. This should *only* be used for development or testing, not for production.
 
-### Setting up a PostgreSQL service
+Paid services may not be enabled for your organisation. If they're not enabled, when you try to set up a paid service, you'll receive the error "service instance cannot be created because paid service plans are not allowed". One of your [Org Managers](/#org-manager) must contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to request that we enable paid services.
 
-To create a service and bind it to your app:
 
-1. From the command line, run:
+### High availability plans
 
-    ``cf marketplace``
+We recommend you use one of the high availability plans (indicated by `HA` in the name) for your production apps. These plans use Amazon RDS Multi-AZ instances which are designed to be 99.95% available (see [Amazon's SLA](https://aws.amazon.com/rds/sla/) for details).
 
-    to see the available services.
+When you use the high availability plan, Amazon RDS provides a hot standby service for failover in the event that the original service fails.
 
-    You will see output like this:
+The failover process means that Amazon RDS will automatically change the DNS record of the database instance to point to the standby instance. You should make sure that your app doesn't cache the database IP, and any DNS caching should be configured with a low time to live (TTL). Consult the documentation for the language/framework you are using to find out how to do this.
 
-    
-        service     plans                                   description
-        postgres    S-dedicated-9.5*, S-HA-dedicated-9.5*,  AWS RDS PostgreSQL service
-                    M-dedicated-9.5*, M-HA-dedicated-9.5*,
-                    L-dedicated-9.5*, L-HA-dedicated-9.5*
+During failover, there will be an outage period (from tens of seconds to a few minutes).
+
+See the [Amazon RDS documentation on the failover process](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.MultiAZ.html#Concepts.MultiAZ.Failover) for more details.
 
-2.  Run:
+You should test how your app deals with a failover to make sure you are benefiting from the high availability plan. We can trigger a failover for you. Please contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) to arrange this.
+
+### Encrypted PostgreSQL plans
+
+Plans with ``enc`` in the name include encryption at rest of the database storage. This means that the data is encrypted while the service is stopped.
+
+We recommend that you use an encrypted plan for production services.
+
+Once you've created a service instance, you can't enable or disable encryption. There's no way to convert an unencrypted PostgreSQL service instance to an encrypted one later.
```

Comment:
Following GOV.UK style, we avoid using bold for emphasis. When callout boxes as used on GOV.UK are added to the template, I'll add a warning callout here ( https://github.com/alphagov/govspeak/wiki/Using-govspeak-on-GOV.UK#warning-callouts)